### PR TITLE
test: Generate test name updates rather than calculate dynamically

### PR DESCRIPTION
**Pull request by smarterclayton**
It is expensive to perform 100 checks against 1k names on every startup.
Instead, move these to generated files that can be reviewed out of bounds
on change. Add them to generated-bindata for verification and regen.

### CODE REVIEW

File: test/extended/util/annotate/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+reviewers:
+  - smarterclayton
+approvers:
+  - smarterclayton
+  - decarr
```
Comment (derekwaynecarr) [inline test/extended/util/annotate/OWNERS]: derekwaynecarr

**Comment by smarterclayton** [comment]
Merging to reduce CPU use and unblock David.

**Pull request merged**